apps/angular-example/src/app/lazy/lazy.component.ts

```diff
@@ -3,16 +3,10 @@ import { Abby } from '../abby';
 
 @Component({
   selector: 'app-lazy',
-  template: '<div>angularTest = {{angularTest}}</div>',
+  template: '<div>angularTest = {{ angularTest | async}}</div>',
```

Comment:
maybe add a singla use-case too

Reply:
We would first need an update to angular 16, I guess this would go beyond the scope of this PR.
Or should I just put a bunch of changes in this one?

Reply:
dunno what the current task it. but yeah, this dedicated one could be a seperate one :+1: